examples/sample-app/README.md

```diff
@@ -205,13 +202,13 @@ the present working directory is the same directory as this README.
     Note that no build has actually occurred yet, so at this time there
     is no image to deploy and no application to visit.
 
-14. Trigger an initial build of your application
+13. Trigger an initial build of your application
  * If you setup the GitHub webhook, push a change to app.rb in your ruby sample repository from step 10.
```

Comment:
step 9.  or just remove the exact step reference, everyone misses this one when renumbering the steps :)